app-cdr/iat/iat-0.1.7-r1.ebuild

```diff
@@ -1,15 +1,15 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
If you update EAPI, **then** you should revbump and drop KEYWORDS. However I'd suggest against that. Just fix HOMEPAGE and SRC_URI.

Reply:
Alright, that's actually what I did originally. Is there a reason why you suggest against it? Already done.

Reply:
Because it requires a revbump and resetting KEYWORDS. That causes more work for stabilization team and they are currently a bit busy with python. 

EAPI-6 is still perfectly valid and works. I don't see a reason to update EAPI here, especialy when it has nothing to do with original intent of this PR (homepage, SRC_URI).